packages/wrangler/src/__tests__/index.test.ts

```diff
@@ -399,18 +435,18 @@ describe("wrangler", () => {
       expect(fs.existsSync("./src/index.js")).toBe(false);
       expect(fs.existsSync("./src/index.ts")).toBe(true);
 
-      expect(packageJson.scripts.start).toBe("wrangler dev src/index.ts");
-      expect(packageJson.scripts.publish).toBe("wrangler publish src/index.ts");
+      expect(packageJson.scripts.start).toBe("wrangler dev");
+      expect(packageJson.scripts.publish).toBe("wrangler publish");
       expect(packageJson.name).toContain("wrangler-tests");
       expect(packageJson.version).toEqual("0.0.0");
       expect(std.out).toMatchInlineSnapshot(`
-              "✨ Successfully created wrangler.toml
-              ✨ Created package.json
-              ✨ Created tsconfig.json, installed @cloudflare/workers-types into devDependencies
-              To start developing on your worker, run npm start.
-              To publish your worker on to the internet, run npm run publish.
-              ✨ Created src/index.ts"
-            `);
+        "✨ Successfully created wrangler.toml
+        ✨ Created package.json
+        ✨ Created tsconfig.json, installed @cloudflare/workers-types into devDependencies
+        ✨ Created src/index.ts
+        To start developing your Worker, run \`npm start\`
+        To publish your Worker to the internet, run \`npm run publish\`"
```

Comment:
```suggestion
        To publish your Worker to the Internet, run \`npm run publish\`"
```

Reply:
It's a Cloudflare thing that we use `Internet` instead of the (technically) correct `internet`.

Reply:
Oh TIL! replacing across the codebase